flink-core/src/main/java/org/apache/flink/configuration/ConfigUtils.java

```diff
@@ -155,5 +155,23 @@ public static Set<ConfigOption<?>> getAllConfigOptions(Class<?> configOptionsCla
         return options;
     }
 
+    /**
+     * Merge pipeline jars from current {@link Configuration} and other into configuration.
+     *
+     * @param jars the jars to be merged to configuration
+     * @param configuration the configuration object to get the pipeline.jars and merge the jars to
+     *     it
+     */
+    public static void mergePipelineJarsToConfig(Set<String> jars, Configuration configuration) {
```

Comment:
Can we abstract it to a common method like `mergeCollectionsToConfig(Set<T> jars, Configuration configuration, final FunctionWithException<T, T, E> mapper)`